src/default-settings.js

```diff
@@ -1,167 +1,177 @@
-const settings = {
-    //Addition settings for this template
-    id_col: 'USUBJID',
-    seq_col: 'AESEQ',
-    soc_col: 'AEBODSYS',
-    term_col: 'AETERM',
-    stdy_col: 'ASTDY',
-    endy_col: 'AENDY',
-    sev_col: 'AESEV',
-    rel_col: 'AEREL',
-    ser_col: 'AESER',
-    filter_cols: [],
-    detail_cols:[],
-
-    //Standard webcharts settings
-    x:{
-        "label":null,
-        "type":"linear",
-        "column":'wc_value'
-    },
-    y:{
-        "column":null, //set in syncSettings()
-        "label": '', 
-        "sort":"earliest",
-        "type":"ordinal",
-        "behavior": 'flex'
-    },
-    "marks":[
-        {
-            "per":null, //set in syncSettings()
-            "tooltip":null, //set in syncSettings()
-            "type":"line",
-            "attributes":{'stroke-width': 5, 'stroke-opacity': .8 },
-        },
-        {
-            "per":null, //set in syncSettings()
-            "tooltip":null, //set in syncSettings()
-            "type":"circle",
-        },
-        {
-            "per":null, //set in syncSettings()
-            "tooltip":null, //set in syncSettings()
-            "type":"line",
-            "attributes":{'stroke-width': 3, 'stroke-opacity': .8, 'stroke': 'black'},
-            "values":{"AESER": ["Yes", 'Y']}
-        },
-        {
-            "per":null, //set in syncSettings()
-            "tooltip":null, //set in syncSettings()
-            "type":"circle",
-            "attributes":{'stroke-width': 3, 'stroke': 'black'},
-            "radius":5,
-            "values":{"AESER": ["Yes", 'Y']}
-        }
-    ],
-    "legend":{
-        "mark":"circle", 
-        "label": 'Severity'
-    },
-    "margin": {"top": 50, bottom: null, left: null, right: null},
-    "colors": ['#66bd63', '#fdae61', '#d73027', '#6e016b'],
-    "date_format":"%m/%d/%y",
-    "resizable":true,
-    "max_width":1000,
-    "y_behavior": 'flex',
-    "gridlines":"y",
-    "no_text_size":false,
-    "range_band":15,
-    "color_by":null //set in syncSettings()
+import clone
+    from './util/clone';
+
+const settings =
+
+  //Template-specific settings
+    {stdy_col: 'ASTDY'
+    ,endy_col: 'AENDY'
```

Comment:
All my insane formatting gets gets roll'd up and babel'd in the end.  But for me it's a matter of aligning everything that can be logically aligned.  Plus leading commas make it easier to add items and properties and spot syntax errors (arguably): https://gist.github.com/isaacs/357981